lib/SILGen/SILGenProlog.cpp

```diff
@@ -262,19 +262,10 @@ struct ArgumentInitHelper {
         gen.B.createDebugValueAddr(loc, address, {vd->isLet(), ArgNo});
         return;
       }
+      gen.VarLocs[vd] = SILGenFunction::VarLoc::get(argrv.getValue());
+      assert(argrv.getType().isAddress() && "expected inout to be address");
+      gen.B.createDebugValueAddr(loc, argrv.getValue(), {vd->isLet(), ArgNo});
```

Comment:
The BuiltinUnsafeValueBufferType case above passes the unmanaged address to DebugValueAddr. This new default case passes a the argv value directly to DebugValueAddr. Why is there any difference?